esp/src/tsconfig.json

```diff
@@ -22,7 +22,8 @@
         "jsx": "react",
         "lib": [
             "dom",
-            "es2019"
+            "es2019",
+            "es2021.String"
```

Comment:
@GordonSmith This change could possibly just have been replacing es2019 with es2021?

Reply:
There is a "replaceAll" function in one of the Utility files already - this is probably not the right place to bump browser version dependencies...